frontend/src/scenes/insights/filters/BreakdownFilter/BreakdownTag.tsx

```diff
@@ -1,14 +1,18 @@
-import { LemonButton, LemonDivider, LemonInput, LemonTag } from '@posthog/lemon-ui'
+import { LemonButton, LemonDivider, LemonInput, LemonSwitch, LemonTag } from '@posthog/lemon-ui'
 import { useActions, useValues } from 'kea'
 import { PropertyKeyInfo } from 'lib/components/PropertyKeyInfo'
 import { cohortsModel } from '~/models/cohortsModel'
 import { FilterType } from '~/types'
 import { breakdownTagLogic } from './breakdownTagLogic'
 import { isAllCohort, isCohort, isPersonEventOrGroup } from './TaxonomicBreakdownFilter'
+import { Tooltip } from 'lib/components/Tooltip'
+import { IconInfo } from 'lib/components/icons'
 
 type BreakdownTagProps = {
     isHistogramable: boolean
-    setFilters?: (filter: Partial<FilterType>) => void
+    // e.g. example.com and example.com/ should be treated as the same value
+    isURLNormalizeable: boolean
```

Comment:
VERY tiny nit: "URL" is camel-cased in `setNormalizeBreakdownUrl`, but all capitalized here